Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -288,7 +288,10 @@ find_path(SwiftShims_INCLUDE_DIR "shims/module.modulemap" HINTS
   ${SwiftShims_INCLUDE_DIR_HINTS})
 add_library(swiftShims INTERFACE IMPORTED GLOBAL)
 set_target_properties(swiftShims PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${SwiftShims_INCLUDE_DIR}/shims")
+  INTERFACE_INCLUDE_DIRECTORIES "${SwiftShims_INCLUDE_DIR}/..;${SwiftShims_INCLUDE_DIR}/shims")
```

Comment:
I would rather that we split these up rather than join them by `;`. Why do we need to include `${SwiftShims_INCLUDE_DIR}/..`?